steps/python/python.deploy.yml

```diff
@@ -0,0 +1,101 @@
+# python.deploy
+#
+# Deploy Python package to PyPI and optionally to a Docker registry.
+#
+
+parameters:
+  pypiConnector: "testpypi-tomtom-dev"
+  dockerDeploy: true
+  dockerRegistryConnector: "tomtom-docker-registry-bintray"
+  dockerRepo: "python"
+  dockerDeployFile: "Dockerfile"
+
+steps:
+- checkout: self
+  # Allow scripts to access the system token so that the pipeline can push git changes (eg. version bump)
+  persistCredentials: true
+
+- script: |
+    git config --global user.email "44620811+tt-ci@users.noreply.github.com"
+    git config --global user.name "tt-ci"
+  condition: eq(variables['release'], 'true')
+  displayName: "Configure git"
+
+# Checkout branch since by default Azure Pipelines do checkouts of specific commits (detached HEAD)
+# and we need to be at master to be able to push version bump changes.
+# We need to use a powershell script here to extract the proper branch name (eg. 'feature/feat-a' out of 'refs/heads/feature/feat-a')
```

Comment:
Might be worth clarifying that this drops the first two parts of the path (i.e. refs/heads)

Reply:
Isn't the comment explaining that?

Reply:
Technically yes, but it reads more like its matching a pattern for a branch name than simply dropping a static prefix at the start.
Given the line of code isn't easy to read, making it really obvious how it behaves would help someone consuming the script debug issues

Reply:
Updated the comment